client/src/listeners/rooms.ts

```diff
@@ -17,9 +17,16 @@ You should have received a copy of the GNU General Public License
 along with Home Lights.  If not, see <http://www.gnu.org/licenses/>.
 */
 
-import { listen } from 'reduxology';
+import { listen, dispatch } from 'reduxology';
 import { Actions } from '../types';
+import { CreateRoomRequest } from '../common/types';
+import { get, post } from '../util/api';
 
-listen(Actions.CreateRoom, () => {
-  // TODO: make API request to create a new room
+listen(Actions.CreateRoom, async (name: string) => {
+  console.log(`Creating room ${name}`);
```

Comment:
Not a huge fan of leaving console logs in code-- is there a reason you'd like to keep this?

Reply:
Wasn't intentional, will remove